deployment/src/main/java/io/quarkiverse/helm/deployment/HelmChartConfig.java

```diff
@@ -108,6 +108,18 @@ public class HelmChartConfig {
     @ConfigItem
     Map<String, ValueReferenceConfig> values;
 
+    /**
+     * The input folder in which to place the user-defined Helm files. These files will be used as inputs to populate the
+     * generated Helm files.
+     * At the moment, the only supported Helm file is the `values.yaml` file.
+     *
+     * It also supports absolute paths.
+     *
+     * By default, it will use the folder "src/main/helm".
+     */
+    @ConfigItem(defaultValue = "src/main/helm")
+    public String inputDirectory;
```

Comment:
should this be Optional<String> instead and without default? just to make it deliberate because users may already be using src/main/helm for something else

Reply:
This follows the same standard that the Quarkus Kubernetes extension where you can put the initial seed of Kubernetes resources at `src/main/kubernetes`.

Reply:
Ah I see! Thanks for the clarification!